modules/charmed-temporal/variables.tf

```diff
@@ -19,7 +19,7 @@ variable "postgresql" {
   description = "Inputs for postgresql-k8s charm module."
   type = object({
     app_name = optional(string, "postgres")
-    channel  = optional(string, "14/stable")
+    channel  = optional(string, "16/edge")
```

Comment:
please add an issue to this repo to switch to `16/stable` once it is released and add a comment with a reminder to update in the preceding line (so we dont forget)